requirements.txt

```diff
@@ -1,26 +1,26 @@
 blessed==1.20.0
-certifi==2023.5.7
-charset-normalizer==3.2.0
+certifi==2023.7.22
+charset-normalizer==3.3.1
 colorama==0.4.6
 distro==1.8.0
 idna==3.4
-jsonschema==4.19.1
+jsonschema==4.19.2
 Jinja2==3.1.2
 MarkupSafe==2.1.3
-peewee==3.16.2
-prompt-toolkit==3.0.39
+peewee==3.16.3
+prompt-toolkit==3.0.36
 psycopg2-binary==2.9.9
 python-dotenv==1.0.0
 python-editor==1.0.4
 pytest==7.4.2
 pyyaml==6.0.1
-questionary==1.10.0
+questionary==2.0.1
```

Comment:
I am wary of upgrading major version of packages without extensive manual testing. Could we keep questionary at the latest 1.x instead (which also might mean we don't need to upgrade a few other that 2.x depends on?)